packages/spacecat-shared-data-access/src/models/site-candidate.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { hasText, isValidUrl } from '@adobe/spacecat-shared-utils';
+
+import { Base } from './base.js';
+
+export const DEFAULT_UPDATED_BY = 'spacecat';
+
+export const SITE_CANDIDATE_STATUS = {
+  DISCOVERED: 'DISCOVERED', // site candidate is discovered
+  PENDING: 'PENDING', // site candidate notification sent and waiting for human input
```

Comment:
there are two steps between discovered and pending. First one is the "helix check" where we check if the site is an edge delivery site or not. Second one is sending the slack message. The idea is to capture the failure during either of those steps. It might be too specific status to care about, but might have benefits too. Still, I don't have any hard feelings on that. We can ditch that status.

Reply:
is the "failure" recorded in the site candidates table for more introspection? if not it may be superfluous. no strong feelings here though 👍🏻

Reply:
will drop the `DISCOVERED` state and add an `ERROR` state instead